quesma/quesma/config/config_v2.go

```diff
@@ -573,6 +561,29 @@ func (c *QuesmaNewConfiguration) TranslateToLegacyConfig() QuesmaConfiguration {
 	}
 
 END:
+
```

Comment:
Personally, not a great fun of using labels `END` that way. This could be a function/method call instead.